core/scripts/benchmark-plus-extras.js

```diff
@@ -24,7 +24,7 @@ async function runOctane(page) {
   process.stdout.write(`Running Octane...\n`);
   await page.goto('https://chromium.github.io/octane/', {waitUntil: 'networkidle0'});
   await page.waitForSelector('#run-octane');
-  await page.waitForTimeout(5000);
+  await new Promise(r => setTimeout(r, 5000));
```

Comment:
We should avoid arbitrary timeouts, but just trying this to ensure test parity first